gluten-core/src/main/scala/io/glutenproject/expression/UnaryExpressionTransformer.scala

```diff
@@ -215,3 +215,27 @@ case class RandTransformer(
     ExpressionBuilder.makeScalarFunction(functionId, inputNodes, typeNode)
   }
 }
+
+case class GetArrayStructFieldsTransformer(
```

Comment:
Have you met any issue to use the default transformation? If not, maybe we don't need to add a specific transformer for this function.

Reply:
Default transformation is not acceptable for current expr. Because ordinal should be passed as an extra argument in substrait scalar function, at least for CH backend(I'm not sure if it is needed by Velox backend)